src/routes/game/components/GameMenu.vue

```diff
@@ -181,6 +186,9 @@ export default {
         this.$router.push('/');
       }
     },
+    async refresh() {
+      await this.authStore.reconnectSocket();
```

Comment:
I used reconnect Sockets instead of request status because this calls request status anyway, and reconnects the socket if it disconnected.